src/libLookup/Lookup.cpp

```diff
@@ -1601,6 +1599,14 @@ bool Lookup::ProcessSetDSInfoFromSeed(const bytes& message, unsigned int offset,
                     << senderPubKey << " is not in my lookup node list.");
       return false;
     }
+  } else {
+    if ((m_mediator.m_currentEpochNum <= 1) && GUARD_MODE &&
+        !Guard::GetInstance().IsNodeInDSGuardList(senderPubKey)) {
+      LOG_EPOCH(WARNING, m_mediator.m_currentEpochNum,
+                "The message sender pubkey: " << senderPubKey
```

Comment:
should log that `m_currentEpochNum <= 1`

Reply:
Since we use `LOG_EPOCH`, then maybe it's sufficient?  Also, this is just during bootstrap anyway